### DISCUSSION TITLE
Is heavier CPU usage in "Private Mode" ("Mode 2") normal?

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#6547] [General]
I noticed that there are now two modes for "Private Mode" ("Mode 1" and "Mode 2"). It's a really cool feature. When doing tests I noticed that in "Mode 2" the CPU usage increases significantly on the remote side (server), going from 1 or 2 percent to around 8 percent and constantly (in "Mode 1" I didn't notice any increase in CPU usage... just normal RustDesk load). I would like to know if this extra CPU usage in "Mode 2" is normal or is it some problem that has not yet been identified. It's okay if it's normal, due to the characteristics of this resource. However, if it is not normal, here is a suggestion for you to take a look.

**Comment by rustdesk** [comment]
@fufesou

**Comment by fufesou** [comment]
@Jemmota Thanks for your kind feedback.

The cpu usage is not very different for me.

<img width="197" alt="1701182361212" src="https://github.com/rustdesk/rustdesk/assets/13586388/ce7813fc-bae1-4032-a61a-e7483bad188f">


https://github.com/rustdesk/rustdesk/assets/13586388/ccbed441-a701-4efd-a456-94cda9e501c6



https://github.com/rustdesk/rustdesk/assets/13586388/350dac93-3594-45c8-b409-75657567c87a


https://github.com/rustdesk/rustdesk/assets/13586388/3394f538-fc71-4adb-b27f-7340de29d0ad

**Reply by fufesou** [reply]
I'll test more if someone else reports this issue.

**Comment by ghost** [comment]
Dear @fufesou, I saw that in your tests everything seems fine with CPU usage. Doing new tests I noticed something curious. At the beginning of my tests everything seemed really OK, CPU usage did not increase. This left me a little confused. So I decided to change the output from my monitor to the computer I was controlling (I had done this previously, when I identified this behavior). I have an LG 4K monitor and I have two computers connected to it (HDMI1 and DisplayPort). When I switched to the DisplayPort output, which is where the computer I'm controlling (server) is, at that moment the CPU jumped from around 1 or 2 percent to the 8 percent I had mentioned. Maybe this is something that only happens here with me, I don't know. Anyway, thank you very much for your attention.
